packages/wrangler/src/dev/dev.tsx

```diff
@@ -247,7 +247,7 @@ function DevSession(props: DevSessionProps) {
 	useCustomBuild(props.entry, props.build);
 
 	const directory = useTmpDir();
-	const handleError = useErrorHandler();
+	useErrorHandler();
```

Comment:
I'm guessing we still need to call this for the side effect

Reply:
I think in this case it's just an orphaned variable.
See the [source](https://github.com/bvaughn/react-error-boundary/blob/v3.1.4/src/index.tsx#L161-L166C2)
and [here](https://github.com/cloudflare/workers-sdk/commit/28cd7361a6386913b62389705c335dd1b12d1dd6#diff-0daa0c9e538e464631807e0cc2ab0d74c0db46f2f646ac81e27e63111fcfcc65R239) where it was initially introduced to handle D1 shims which have since been removed.

Reply:
cool, removed